pyproject.toml

```diff
@@ -1,13 +1,13 @@
 [tool.poetry]
 name = "alinka-pyside"
-version = "0.3.0"
+version = "0.3.1"
 description = "Application for Supporting Center for Children"
 authors = ["CodeForPoznań <hello@codeforpoznan.pl>"]
 
 [tool.poetry.dependencies]
 Jinja2 = "^3.1.2"
 pydantic = "^1.10.2"
-pyside6 = "^6.5.0"
+pyside2 = "^5.15.2.1"
```

Comment:
Why do you want to downgrade it? I don’t get from this code why the downgrade was needed. But maybe I am missing something. ;-)

Reply:
I don't remember neither. It might have been some packages and python version mismatch. But - to be straight - I don't remember. I'd leave as it is, and bump later - if needed.